### ISSUE TITLE
Setup instructions from source with AutoHotkey installed

### ISSUE THREAD

**Issue by jlamoure** [fuhsjr00/bug.n#226]
I cannot install the executable due to work security restrictions, is there a proper way to install from source? I have copied all files to a folder, and run Main.ahk with Autohotkey, but keybindings do not work and I wonder if I have missed an installation step.

Windows 10

**Comment by joten** [comment]
No, there is not much more to it. If you have a file association .ahk -> AutoHotkey and double-click Main.ahk in Explorer, bug.n should run just fine; the directory containing Main.ahk should also be the working directory, but if bug.n started without an error, this should be the case.

* Is the icon visible in the system tray and can you open the context menu?
* Does any hotkey work?
* Does the dynamic tiling work? Opening e.g. three Explorer windows with the default configuration for bug.n should result in a tiled layout.
* You may try and set different hotkeys, e.g. with the Alt-key `!` as a modifier instead of the Win-key `#`.
